rtbkit/core/router/router_runner.cc

```diff
@@ -105,7 +106,9 @@ doOptions(int argc, char ** argv,
         ("local-banker-debug", bool_switch(&localBankerDebug),
          "enable local banker debug for more precise tracking by account")
         ("banker-choice", value<string>(&bankerChoice),
-         "split or local banker can be chosen.");
+         "split or local banker can be chosen.")
+         ("augmenter-timeout",value<int>(&augmentationWindowms),
+         "configure the augmenter  timeout");
```

Comment:
you should add the unit that the argument accepts in the description. i.e. that the value is in milliseconds.